src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import { Route } from 'react-router-dom';
 
 const App = () => (
   <div>
     <Navigation />
+    <Route exact path="/" component={Home} />
+    <Route exact path="/about" component={About} />
```

Comment:
Exact is only needed for broader more vague links, for example /about would render both the Home and About component (if Home didn't have exact) because /about has a / which would trigger Home and /about would trigger About obviously.